### PR TITLE
Feat/cleaning+refactor

### PR DESCRIPTION
**Pull request by cavacado**
- refactored `package json` and introduced different config files
- remove `enzyme` dependency / refactored tests that use `enzyme` / configurations relating to `enzyme`, also remove `sinon`
- remove unnecessary npm modules 
- remove `sendCertificate` and all code where it is being consumed (including reducers)
~~- change to importing lodash modules rather than top-level lodash as we are using it in 2 instances only~~
- get rid of dangling assets / references from old (open-certs time?)
- introduce `husky` for precommits (when commiting, it will run lint before commiting changes)

### CODE REVIEW

File: src/components/UI/Textarea/Textarea.tsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
 
 export const TextareaDefault = styled.textarea`
-  ${StyledInput()}
+  ${tw`border border-solid border-gray-300 bg-white py-1 px-2 mb-2`}
+  &::placeholder {
+    ${tw`italic text-gray-500 text-base`}
+  }
   min-height: 140px;
   width: 100%;
   resize: none;
```
Comment (simboonlong) [inline src/components/UI/Textarea/Textarea.tsx]: maybe let's delete this and its storybook totally and use in `EmailForm` directly (it's only used there)

```
const TextArea = styled.textarea`
  min-height: 140px;
  &::placeholder {
    ${tw`italic text-gray-500 text-base`}
  }
`;

<TextArea className="border border-solid border-gray-300 bg-white py-1 px-2 mb-2 w-full resize-none" />
```

i not sure if u can do `@apply <tailwindclass>`, if can best. if cannot then leave it to tw macro for the placeholder styles i guess

File: src/tailwind.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const _ = require("lodash"); //eslint-disable-line @typescript-eslint/no-var-requires
+const merge = require("lodash/merge"); //eslint-disable-line @typescript-eslint/no-var-requires
```
Comment (simboonlong) [inline src/tailwind.js]: think no longer need eslint-disable

File: src/components/UI/Textarea/Textarea.tsx (hunk 1 above)
Comment (cavacado) [reply src/components/UI/Textarea/Textarea.tsx]: yup ok, i'll do the suggested change, not too sure about `@apply <tailwindclass>`, is that tailwind's way of mix-ins?

File: src/tailwind.js (hunk 2 above)
Comment (cavacado) [reply src/tailwind.js]: 👌

**Review by isaackps — COMMENTED** [review]
LGTM!

**Review by isaackps — APPROVED** [review]

File: src/components/UI/Textarea/Textarea.tsx (hunk 1 above)
Comment (simboonlong) [reply src/components/UI/Textarea/Textarea.tsx]: um u can use it in `tailwind.css` that file there, but internally i not sure if they will magic it not

**Pull request merged**